modules/docs/mdx/5.0-MIGRATION-GUIDE.mdx

```diff
@@ -69,3 +76,25 @@ Components are transitioning to [Compound Components](./COMPOUND_COMPONENTS.md)
 PR:
 
 - https://github.com/Workday/canvas-kit/pull/953
+
+### Button
+
+- `beta_Button` export was removed.
```

Comment:
Shouldn't the codemod handle this too? 🤖

Reply:
Yes. Definitely. I see 102 instances on GHE alone

Reply:
All cases I could find looked like:

```tsx
import { beta_Button as Button } from '@workday/canvas-kit-react-button;
```

Reply:
The previous will now be rewritten as:
```tsx
import { Button } from '@workday/canvas-kit-react-button;
```

Also renaming is supported:
```tsx
// before
import { beta_Button as BetaButton } from '@workday/canvas-kit-react-button';

// after
import { Button as BetaButton } from '@workday/canvas-kit-react-button';
```